metadata-ingestion/src/datahub/ingestion/source/looker/lookml_source.py

```diff
@@ -795,7 +795,12 @@ def get_internal_workunits(self) -> Iterable[MetadataWorkUnit]:  # noqa: C901
 
             for include in model.resolved_includes:
                 logger.debug(f"Considering {include} for model {model_name}")
-                if include.include in processed_view_files:
+                if (
+                    include.include in processed_view_files
+                    and "model" not in self.source_config.view_naming_pattern.pattern
```

Comment:
Yes. But it will only allow multiple view files to be let through since a view file can be shared between model files belonging to the same connection. There will not be duplicate entries of the view file with a model file since it will be caught later in the script with the `view_connection_map`. Let me know what you think

Reply:
we should separate the `"model" in pattern` part from the actual disabling of the view deduplication logic

also, we'll need a significantly more detailed comment explaining why it's safe to do and how it impacts other parts of the code

Reply:
Yeah I agree with that. I thought only disabling it for when {model} is in the URN. Because only then we will have the issue with same view file, for different models, with same connection not being processed.